logging/cluster-logging-dashboards.adoc

```diff
@@ -6,11 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The *Logging/Elasticsearch Nodes* and *Openshift Logging* dashboards in the {product-title} web console show in-depth details about your Elasticsearch instance and the individual Elasticsearch nodes that you can use to prevent and diagnose problems.
+The *Logging/Elasticsearch Nodes* and *{logging}* dashboards in the {product-title} web console show in-depth details about your Elasticsearch instance and the individual Elasticsearch nodes that you can use to prevent and diagnose problems.
```

Comment:
Are you sure the dashboard labels in the UI are going to be say 'logging subsystem for Red Hat OpenShift'?

Reply:
This one shouldn't have been changed, good catch.

Reply:
Same for the next two instances...